src/components/Generations.js

```diff
@@ -0,0 +1,29 @@
+import './style/Generations.css';
+import Kanto from './regions/Kanto';
+import Johto from './regions/Johto';
+import Hoenn from './regions/Hoenn';
+import Sinnoh from './regions/Sinnoh';
+import Teselia from './regions/Teselia';
+import Kalos from './regions/Kalos';
+import Alola from './regions/Alola';
+import Galar from './regions/Galar';
+
+const Generations = () => (
```

Comment:
- [ ] The Home page should have a list of categories that could be filtered using given parameters, e.g. category name, kindly consider adding a filter option to the home page this could be search functionality to filter pokemon region by name for example